src/parser/jobs/pld/modules/FightOrFlight.tsx

```diff
@@ -109,34 +46,50 @@ export class FightOrFlight extends BuffWindow {
 		this.ignoreActions(EXCLUDED_ACTIONS.map(g => this.data.actions[g].id))
 
 		this.addEvaluator(new ExpectedGcdCountEvaluator({
-			expectedGcds: 11,
+			expectedGcds: 8,
 			globalCooldown: this.globalCooldown,
 			suggestionIcon: this.data.actions.FIGHT_OR_FLIGHT.icon,
 			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcds.content">
-				Try to land 11 physical GCDs during every <DataLink action="FIGHT_OR_FLIGHT" /> window.
+				Try to land 8 GCDs during every <DataLink action="FIGHT_OR_FLIGHT" /> window.
 			</Trans>,
 			suggestionWindowName,
 			severityTiers: SEVERITIES.MISSED_GCD,
 		}))
 
+		this.addEvaluator(new ExpectedActionsEvaluator({
+			expectedActions: [
+				{action: this.data.actions.GORING_BLADE, expectedPerWindow: 1},
+				{action: this.data.actions.CONFITEOR, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_FAITH, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_TRUTH, expectedPerWindow: 1},
+				{action: this.data.actions.BLADE_OF_VALOR, expectedPerWindow: 1},
+				{action: this.data.actions.HOLY_SPIRIT, expectedPerWindow: 1},
+			],
+			suggestionIcon: this.data.actions.EXPIACION.icon,
+			suggestionContent: <Trans id="pld.fightorflight.suggestions.gcd_actions.content">
+				Try to land at least one cast of <DataLink action="GORING_BLADE" />
```

Comment:
gunna ditto myself about action lists in suggestions

Reply:
I ditto my comment above.